packages/superset-ui-core/src/SupersetClient.js

```diff
@@ -75,14 +75,15 @@ class SupersetClient {
     );
   }
 
-  get({ host, url, endpoint, mode, credentials, headers, body, timeout, signal }) {
+  get({ body, credentials, headers, host, endpoint, mode, parseMethod, signal, timeout, url }) {
```

Comment:
It starts to get long, do you want to separate into one param per line?

Reply:
Oh just notice `post` is already doing that, then maybe good to spread this one out too.

Reply:
it's all based on prettier config, it auto wraps if it's over the 100 char limit (which `post` is but `get` is not). we could decrease the char limit if you feel strongly about it tho.

Reply:
nope. this is pretty minor. I'll leave it to you.